rpc/module.go

```diff
@@ -0,0 +1,74 @@
+package rpc
+
+import (
+	"fmt"
+	"log"
+
+	// importing because used by code-generated files that are git ignored and to allow go mod tidy and go mod vendor to function properly
+	_ "github.com/getkin/kin-openapi/openapi3"
+	_ "github.com/labstack/echo/v4"
+
+	"github.com/pokt-network/pocket/shared/modules"
+)
+
+var (
+	_ modules.RPCModule = &rpcModule{}
+)
+
+type rpcModule struct {
+	bus    modules.Bus
+	config modules.RPCConfig
+}
+
+const (
+	RPCModuleName = "rpc"
```

Comment:
since we're implementing InitializableModule, this const does not need to be public. Arguably it shouldn't be?

Reply:
good catch, applied consistently across the board https://github.com/pokt-network/pocket/pull/176/commits/9958b01c1d442c1affdd7f4679711a5488b43552